benchmarks/corpus/README.md

```
@@ -1,29 +1,69 @@
-A robustness harness that runs docx-editor over a 56-file corpus of real-world
-`.docx` files from 14 producer flavors (Word, LibreOffice, pandoc, ONLYOFFICE,
-and the test suites of python-docx, mammoth, Apache POI, pandoc, and
-LibreOffice core).
+A robustness harness that runs docx-editor over a 77-file corpus of real-world
+`.docx` files from 15 producer flavors (Word, LibreOffice, pandoc, ONLYOFFICE,
+and the test suites of python-docx, mammoth, Apache POI, pandoc, LibreOffice
+core, and Open-Xml-PowerTools).
 
 Each file goes through the stages:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)